ca/config_test.go

```diff
@@ -264,17 +266,22 @@ func TestLoadSecurityConfigIntermediates(t *testing.T) {
 	rootCA, err := ca.NewRootCA(testutils.ECDSACertChain[2], nil, nil, ca.DefaultNodeCertExpiration, nil)
 	require.NoError(t, err)
 
+	ctx := log.WithLogger(context.Background(), log.L.WithFields(logrus.Fields{
```

Comment:
Is this supposed to be `t.Context`? I only ask because elsewhere this PR changes a lot of `context.Background()` into `t.Context`. Maybe it doesn't matter much for the logger?